stable/fluentd-elasticsearch/templates/daemonset.yaml

```diff
@@ -50,6 +50,10 @@ spec:
           value: {{ .Values.elasticsearch.logstash_prefix | quote }}
         - name: OUTPUT_SCHEME
           value: {{ .Values.elasticsearch.scheme | quote }}
+        {{- if eq .Values.elasticsearch.scheme "https" }}
```

Comment:
sorry, lets remove the if statement again please.
otherwise we have an empty tls setting with http scheme